src/keyboard.rs

```diff
@@ -69,7 +69,7 @@
 //
 // --------- END OF W3C SHORT NOTICE ---------------------------------------------------------------
 
-use smol_str::SmolStr;
+pub use smol_str::SmolStr;
```

Comment:
Could you inline docs?

Reply:
Looks fine to me when generated locally

Reply:
Maybe it works due to the `pub use` and not because it's an alias from the `SmolStr`, I know that docs are not inlined in some cases.

Reply:
Half the time it seems like rustdoc completely ignores inline/no_inline doc attributes; I could never figure it out.